src/program_generator/data/selection.csv

```diff
@@ -51,6 +51,12 @@ Opcode Name
 0x51 MLOAD
 0x52 MSTORE
 0x53 MSTORE8
+0x54 SLOAD_COLD
+0x54 SLOAD_WARM
+0x55 SSTORE_COLD_CHANGE
+0x55 SSTORE_COLD_NO_CHANGE
+0x55 SSTORE_WARM_CHANGE
+0x55 SSTORE_WARM_NO_CHANGE
```

Comment:
```suggestion
0x54 SLOAD
0x55 SSTORE
```

Reply:
@lukasz-glen why do we want to remove it from the default selection file? And please note there are no `SLOAD` and `SSTORE` in `opcodes.csv`